### PR TITLE
[PAN-2117] Error when removing bootnodes from nodes whitelist

### PR DESCRIPTION
**Pull request by lucassaldanha**
## PR description
Preventing user from removing a bootnode from the nodes whitelist

### CODE REVIEW

File: ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/permissioning/PermRemoveNodesFromWhitelist.java [hunk 1]
```diff
@@ -68,6 +68,9 @@ public JsonRpcResponse response(final JsonRpcRequest req) {
               return new JsonRpcErrorResponse(req.getId(), JsonRpcError.WHITELIST_PERSIST_FAILURE);
             case ERROR_WHITELIST_FILE_SYNC:
               return new JsonRpcErrorResponse(req.getId(), JsonRpcError.WHITELIST_FILE_SYNC);
+            case ERROR_CANT_REMOVE_BOOTNODE:
```
Comment (macfarla) [inline ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/permissioning/PermRemoveNodesFromWhitelist.java]: I don't love having the contraction in the constant name - do we have a convention on this? maybe change to cannot?

Comment (macfarla) [reply ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/permissioning/PermRemoveNodesFromWhitelist.java]: or maybe use "prevent"

Comment (macfarla) [reply ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/permissioning/PermRemoveNodesFromWhitelist.java]: ERROR_PREVENT_BOOTNODE_REMOVAL ?

Comment (lucassaldanha) [reply ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/permissioning/PermRemoveNodesFromWhitelist.java]: I think I'd rather use ERROR_CANNOT_REMOVE_BOOTNODE. Is that ok?

Comment (lucassaldanha) [reply ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/permissioning/PermRemoveNodesFromWhitelist.java]: `ERROR_BOOTNODE_CANNOT_BE_REMOVED`

**Pull request merged**